JS/projects.js

```diff
@@ -0,0 +1,89 @@
+// // Projects section
+
+export const projectsarray = [
+  {
+    Name: 'Multi-Post Stories Gain+Glory',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s./",
+    img: './assests/images/project1-img.svg',
```

Comment:
- [ ] Your images are still not displaying after cloning and recreating the project. This issue is persistent across the mobile and desktop versions of the popup. Kindly consider the image source if its a problem, and make sure you confirm before sending it back for review:
- The above has made is not possible to check for the changes asked by the last reviewer 

### Blank images Mobile:
![image](https://github.com/GeekyHacks/My-Personal-Portfolio/assets/50990925/96dbb549-eab0-4952-af34-60a45e444fbc)

### Blank image desktop:
![image](https://github.com/GeekyHacks/My-Personal-Portfolio/assets/50990925/be5838e8-4c1d-4cbf-b850-17b5e64fd50a)

Reply:
It appears but it takes time to load, I would appreciate suggesting a solution for that.

Reply:
https://github.com/GeekyHacks/My-Personal-Portfolio/assets/86020057/04fc0ce5-5553-4657-9715-c898d5b5af0e

Please view the video.